pkg/operator/ceph/nfs/controller.go

```diff
@@ -132,6 +142,11 @@ func (c *CephNFSController) onUpdate(oldObj, newObj interface{}) {
 }
 
 func (c *CephNFSController) onDelete(obj interface{}) {
+	if c.clusterSpec.External.Enable && !c.clusterSpec.External.RunStatelessWorkload {
+		logger.Warningf("Deleting nfs for an external ceph cluster is disabled (RunStatelessWorkload is false)")
```

Comment:
Nit: from the user-facing side of things, this is `runStatelessWorkload` not capitalized.